tests/unit/test_charm.py

```diff
@@ -344,13 +344,51 @@ def bundle_handler(args: ops.testing.ExecArgs) -> None:
     charm: DiscourseCharm = typing.cast(DiscourseCharm, harness.charm)
 
     email = "sample@email.com"
-    password = "somepassword"  # nosec
     event = MagicMock(spec=ActionEvent)
     event.params = {
         "email": email,
-        "password": password,
     }
-    charm._on_add_admin_user_action(event)
+    charm._on_promote_user_action(event)
+
+
+def test_create_user():
+    """
+    arrange: an email and a password
+    act: when the _on_create_user_action method is executed
+    assert: the underlying rake command to add the user is executed
+        with the appropriate parameters.
+    """
+    harness = helpers.start_harness()
+
+    # We catch the exec call that we expect to register it and make sure that the
+    # args passed to it are correct.
+    expected_exec_call_was_made = False
```

Comment:
Is this variable used? I cannot see the asset that checks that this variable is true.

Reply:
hmm, seems like previous tests did not assert this variable. i have made sure to assert this now.